include/gz/math/Inertial.hh

```diff
@@ -123,6 +166,52 @@ namespace gz
         return R * this->massMatrix.Moi() * R.Transposed();
       }
 
+      /// \brief Spatial mass matrix for body B. It does not include fluid
+      /// added mass.
+      /// The matrix is arranged as follows:
+      ///
+      /// | m          0          0          0           m * Pz    -m * Py |
+      /// | 0          m          0          -m * Pz    0           m * Px |
+      /// | 0          0          m           m * Py    -m * Px    0       |
+      /// | 0          -m * Pz     m * Py    Ixx        Ixy        Ixz     |
+      /// |  m * Pz    0          -m * Px    Ixy        Iyy        Iyz     |
+      /// | -m * Py     m* Px     0          Ixz        Iyz        Izz     |
+      ///
+      /// \return The body's 6x6 inertial matrix.
+      /// \sa SpatialMatrix
+      public: Matrix6<T> BodyMatrix() const
+      {
+        Matrix6<T> result;
+
+        result.SetSubmatrix(Matrix6<T>::TOP_LEFT,
+            Matrix3<T>::Identity * this->massMatrix.Mass());
+
+        result.SetSubmatrix(Matrix6<T>::BOTTOM_RIGHT, this->massMatrix.Moi());
```

Comment:
`this->massMatrix.Moi()` is expressed in frame `Bi`, but this API should return a matrix in frame `F`
```suggestion
        result.SetSubmatrix(Matrix6<T>::BOTTOM_RIGHT, this->Moi());
```

Reply:
Thanks. While implementing this I went back and forth between the frames, but I agree that the inertial class' API should consistently return values that take the pose into account. Updated in deac75cebb3a89576101407e79bb9aac853766b7.